src/Components/PresentationalComponents/CVEPageDetailsDescription/CVEPageDetailsExpandable.js

```diff
@@ -0,0 +1,49 @@
+import { Text, TextContent } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+import './CVEPageDetailsDescription.scss';
+import { DESCRIPTION_CHARACTER_LIMIT } from '../../../Helpers/constants';
+
+const CVEPageDetailsExpandable = props => {
+    const [isDescriptionTruncated, toggleDescriptionTruncated] = useState(true);
+    const [isDescriptionTooLong, toggleDesciptionTooLong] = useState(false);
+    const [truncatedDescription, setTruncatedDescription] = useState('');
+    const description = props.content;
+
+    useEffect(() => {
+        if (String(props.content).length > DESCRIPTION_CHARACTER_LIMIT) {
+            toggleDesciptionTooLong(true); 
+            setTruncatedDescription(props.content.slice(0, DESCRIPTION_CHARACTER_LIMIT - 1) + '…');
+        }
+        else {
+            toggleDesciptionTooLong(false); 
+        }
+    }, []);
+
+    const toggleDescription = () => {
+        toggleDescriptionTruncated(!isDescriptionTruncated);
+    }
+
+    if (isDescriptionTooLong) {
+        return (
+            <TextContent>
+                <Text component={props.component}>{isDescriptionTruncated ? truncatedDescription : description}</Text>
+                <a rel="noopener noreferrer" onClick={toggleDescription}> {isDescriptionTruncated ? 'Read more' : 'Read less'} </a>
```

Comment:
`Read more` and `Read less` can be the default displayed text but to make it more flexible it's better to pass them as props.